hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -354,8 +354,8 @@ private static EthTxData populateLegacyEthTxData(RLPItem rlpItem, byte[] rawTx)
         }
 
         byte[] chainId = null;
-        byte[] v = rlpList.get(6).asBytes();
-        BigInteger vBI = new BigInteger(1, v);
+        byte[] val = rlpList.get(6).asBytes();
+        BigInteger vBI = new BigInteger(1, val);
```

Comment:
`val` has a meaning in some contexts (e.g. lombok). We may want something different here.

Reply:
I believe lombock uses it as the parameter name for the generated setters, but there should be no conflict here. We may choose to avoid using `val` and instead use `value` or some other variable name. 

If there is a more specific and meaningful name for this variable, then it would be preferable to use it over a generic name. I didn't take the time to understand the code fragment well enough to come with a better name, but I am open to changing it if there are other suggestions.